libs/ais-service/src/main/java/uk/gov/di/ipv/core/library/ais/enums/AisAuditLevel.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.di.ipv.core.library.ais.enums;
+
+// Suppress enums should be upper case rule. These have to match the values used on the AIS API
+@SuppressWarnings("java:S115")
+public enum AisAuditLevel {
+    standard,
+    enahnced
```

Comment:
```suggestion
    enhanced
```

Could this not be put into the Dto class?

Reply:
It could, but it is used in the TestData class and this is more in line with the other services we have
It's marginal, but I think the DTO class is already a bit big with all the subclasses